### PR TITLE
All changes consolidated

### PR DESCRIPTION
**Pull request by milidoshi26**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -100,13 +106,19 @@ const executeDockerCommand = async (dockerCommand: string, continueOnError: bool
     var execOptions: any = {
         outStream: new NullOutstreamStringWritable({ decodeStrings: false }),
         listeners: {
-            stdout: (data: any) => console.log(data.toString()), //to log the script output while the script is running.
-            errline: (data: string) => {
+            stdout: async (data: any) => {
+                let scannedResult = {result: null};
+                let temporaryVariable = await cs.credscan(data.toString(), scannedResult, 1);
```
Comment (vineetmimrot) [inline src/main.ts]: temporaryVariable is not used,

File: src/main.ts [hunk 2]
```diff
@@ -100,13 +106,19 @@ const executeDockerCommand = async (dockerCommand: string, continueOnError: bool
...
+                console.log(scannedResult.result);
+            }, //to log the script output while the script is running.
+            errline: async (data: string) => {
                 if (!shouldOutputErrorStream) {
                     errorStream += data + os.EOL;
                 }
                 else {
-                    console.log(data);
+                    let scannedResult = {result: null};
+                    let temporaryVariable = await cs.credscan(data, scannedResult, 1);
```
Comment (vineetmimrot) [inline src/main.ts]: ditto

File: src/main.ts [hunk 3]
```diff
@@ -32,6 +33,11 @@ const run = async () => {
             core.setFailed('Please enter a valid script.');
             return;
         }
+        console.log(`Starting CredScan.`);
+        let credscancheck: string = inlineScript;
```
Comment (vineetmimrot) [inline src/main.ts]: what is the need for this initialization?

Comment (milidoshi26) [reply src/main.ts]: I'll remove it

File: src/main.ts (hunk 1 above)
Comment (milidoshi26) [reply src/main.ts]: Removed the assignment

File: src/main.ts [hunk 4]
```diff
@@ -32,6 +33,10 @@ const run = async () => {
             core.setFailed('Please enter a valid script.');
             return;
         }
+        console.log(`Starting CredScan.`);
+        let scannedResult = {result: null};
+        let credscancheck: string = await cs.credscan(inlineScript, scannedResult, 1);
```
Comment (vineetmimrot) [inline src/main.ts]: credscancheck -> we are not using it

Comment (milidoshi26) [reply src/main.ts]: Removed assignment

**Pull request closed without merging**